packages/medusa/src/services/cart.js

```diff
@@ -1027,7 +1035,7 @@ class CartService extends BaseService {
 
       // If cart total is 0, we don't perform anything payment related
       if (cart.total <= 0) {
-        cart.completed_at = new Date()
+        cart.payment_authorized_at = new Date()
         return cartRepository.save(cart)
       }
 
```

Comment:
At this point in time the cart os not really completed, which is why we added the new flag.